lib/ecto/adapters/myxql/connection.ex

```diff
@@ -207,6 +207,67 @@ if Code.ensure_loaded?(MyXQL) do
       ["DELETE FROM ", quote_table(prefix, table), " WHERE " | filters]
     end
 
+    @impl true
+    # Opts are deprecated, see Notes at https://dev.mysql.com/doc/refman/5.7/en/explain.html
+    def explain_query(conn, query, _explain_opts, params \\ [], opts \\ []) do
```

Comment:
Let's keep the signature as `def explain_query(conn, query, _explain_opts, params, opts) do` as all arguments are always passed in.